api/v1beta1/machinedeployment_webhook.go

```diff
@@ -156,65 +213,3 @@ func (m *MachineDeployment) validate(old *MachineDeployment) error {
 
 	return apierrors.NewInvalid(GroupVersion.WithKind("MachineDeployment").GroupKind(), m.Name, allErrs)
 }
-
-// PopulateDefaultsMachineDeployment fills in default field values.
-// This is also called during MachineDeployment sync.
-func PopulateDefaultsMachineDeployment(d *MachineDeployment) {
```

Comment:
I think it makes sense to get rid of this exported func as we don't need it anymore.